src/main/java/sirius/db/mixing/OMA.java

```diff
@@ -148,6 +148,13 @@ public boolean isReady() {
             EntityDescriptor ed = entity.getDescriptor();
             ed.beforeSave(entity);
 
+            for (Property property : ed.getProperties()) {
```

Comment:
Nope, just fix EntityRef#containsNonpersistenValue / setValue to handle "new" values properly